pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/integrationLayer/ContentListViewModel.kt

```diff
@@ -108,6 +83,35 @@ class ContentListViewModel(
         }
     }
 
+    /**
+     * Get the URL of the scaled image, in the specified format, to match as much as possible the container width.
+     *
+     * @param imageUrl The original image URL.
+     * @param containerWidth The width, in pixels, of the image container.
+     * @param format The desired format of the transformed image.
+     *
+     * @return xx
+     */
+    fun getScaledImageUrl(
+        imageUrl: String,
+        @Px containerWidth: Int,
+        format: ImageFormat = ImageFormat.WEBP
+    ): String {
+        val size = IlImage.Size.getClosest(containerWidth)
+
+        return imageScalingService.getScaledImageUrl(
+            imageUrl = imageUrl,
+            width = enumValueOf(size.name),
+            format = format
+        )
+    }
+
+    private fun <T : Any, R : Any> Flow<PagingData<T>>.mapPaging(transform: (T) -> R): Flow<PagingData<R>> {
```

Comment:
```suggestion
    private fun <T: Any, R: Any> Flow<PagingData<T>>.mapPaging(transform: (T) -> R): Flow<PagingData<R>> {
```

Reply:
Kotlin uses a space before and after the colon to specify inheritance